administration/src/bp-modules/applications/VerificationsView.tsx

```diff
@@ -16,6 +17,12 @@ const StyledIndicator = styled.span`
   padding: 4px;
 `
 
+const JuleicaIcon = styled.span`
```

Comment:
You can remove this span if you set the style on the parent.
Its better to set the height on the parent and to fit the children inside, so you could add any other assets without setting height to a particular value
Example:
```js
const UnFocusedDiv = styled.div`
  display: flex;
  align-items: center;
  flex: 1;
  :focus {
    outline: none;
  }
  height: 25px;
`
```